### PR TITLE
Add Intel FIT parser

### PR DESCRIPTION
**Pull request by xaionaro**
Adding a parser of Intel FIT metadata structure.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,6 +6,7 @@ require (
 	github.com/dustin/go-humanize v1.0.0
 	github.com/fatih/camelcase v1.0.0
 	github.com/fatih/structtag v1.2.0 // indirect
+	github.com/hashicorp/go-multierror v1.1.1
```
Comment (rihter007) [inline go.mod:9]: Awesome :)

**Review by rihter007 — DISMISSED** [review]

File: pkg/intel/metadata/manifest/common/manifestcodegen/cmd/manifestcodegen/main.go [hunk 2]
```diff
@@ -76,6 +76,10 @@ func processPath(path string, isCheck, enableTracing bool) error {
 		}
 	}
 
+	// ugly terrible hack to workaround versioning support
+	// TODO: fix the importer to recognize `/v2/` as version, not as path
+	os.Setenv("GO111MODULE", "off")
```
Comment (orangecms) [inline pkg/intel/metadata/manifest/common/manifestcodegen/cmd/manifestcodegen/main.go:81]: Having messed enough with Go modules, this is my favourite part, haha :D

**Review by orangecms — APPROVED** [review]
LGTM :)

File: pkg/intel/metadata/manifest/common/manifestcodegen/cmd/manifestcodegen/main.go (hunk 2 above)
Comment (orangecms) [reply pkg/intel/metadata/manifest/common/manifestcodegen/cmd/manifestcodegen/main.go:81]: Oh I see, it was removed, and I brought it back when attemping to resolve the conflicts... sorry, needs to be undone.

**Comment by orangecms** [comment]
This has a lot of conflicts in here; it seems like there were quite different histories, for some reason. Can you take care of that?

**Comment by orangecms** [comment]
Okay, I've done the rebase on a fun Sunday afternoon - fixed up the mess I had created when resolving conflicts earlier.
Could you please recheck? Thank you!

**Review by orangecms — APPROVED** [review]

**Pull request merged**